test/src/test/kotlin/graphql/nadel/tests/next/fixtures/hydration/HydrationThrowsErrorTestSnapshot.kt

```diff
@@ -0,0 +1,100 @@
+// @formatter:off
+package graphql.nadel.tests.next.fixtures.hydration
+
+import graphql.nadel.tests.next.ExpectedNadelResult
+import graphql.nadel.tests.next.ExpectedServiceCall
+import graphql.nadel.tests.next.TestSnapshot
+import graphql.nadel.tests.next.listOfJsonStrings
+import kotlin.Suppress
+import kotlin.collections.List
+import kotlin.collections.listOf
+
+private suspend fun main() {
+    graphql.nadel.tests.next.update<HydrationThrowsErrorTest>()
+}
+
+/**
+ * This class is generated. Do NOT modify.
+ *
+ * Refer to [graphql.nadel.tests.next.UpdateTestSnapshots
+ */
+@Suppress("unused")
+public class HydrationThrowsErrorTestSnapshot : TestSnapshot() {
+    override val calls: List<ExpectedServiceCall> = listOf(
+            ExpectedServiceCall(
+                service = "issues",
+                query = """
+                | {
+                |   issueById(id: "ari:cloud:jira:19b8272f-8d25-4706-adce-8db72305e615:issue/1") {
+                |     id
+                |     key
+                |     hydration__assignee__assigneeId: assigneeId
+                |     __typename__hydration__assignee: __typename
+                |   }
+                | }
+                """.trimMargin(),
+                variables = " {}",
+                result = """
+                | {
+                |   "data": {
+                |     "issueById": {
+                |       "id": "ari:cloud:jira:19b8272f-8d25-4706-adce-8db72305e615:issue/1",
+                |       "key": "GQLGW-1",
+                |       "hydration__assignee__assigneeId": "ari:cloud:identity::user/1",
+                |       "__typename__hydration__assignee": "Issue"
+                |     }
+                |   }
+                | }
+                """.trimMargin(),
+                delayedResults = listOfJsonStrings(
+                ),
+            ),
+        )
+
+    /**
+     * ```json
+     * {
+     *   "errors": [
+     *     {
+     *       "message": "BYE",
+     *       "locations": [],
+     *       "extensions": {
+     *         "classification": "Bye"
+     *       }
+     *     }
+     *   ],
+     *   "data": {
+     *     "issueById": {
+     *       "id": "ari:cloud:jira:19b8272f-8d25-4706-adce-8db72305e615:issue/1",
+     *       "key": "GQLGW-1",
+     *       "assignee": null
+     *     }
+     *   }
+     * }
+     * ```
+     */
+    override val result: ExpectedNadelResult = ExpectedNadelResult(
+            result = """
+            | {
+            |   "errors": [
+            |     {
+            |       "message": "BYE",
+            |       "locations": [],
+            |       "extensions": {
+            |         "classification": "Bye"
+            |       }
+            |     }
+            |   ],
+            |   "data": {
+            |     "issueById": {
+            |       "id": "ari:cloud:jira:19b8272f-8d25-4706-adce-8db72305e615:issue/1",
+            |       "key": "GQLGW-1",
+            |       "assignee": null
```

Comment:
So this part is `null` but before this change the entire `issueById` field was null